lnbits/tasks.py

```diff
@@ -19,6 +19,23 @@
 
 from .core import db
 
+tasks: List[asyncio.Task] = []
+
+
+def create_task(coro_or_fut):
```

Comment:
- name nitpick: change `coro_or_fut` to `coro_or_func` (consistent with the below use and more clear)

Reply:
`coro_or_fut` is the same as asyncio uses themselves. It doesnt have to be the same as below because it isn't, because the below takes a func which creates a coroutine

Reply:
`coro` would be the most accurate in this case